SQL/SQL (basic).sublime-syntax

```diff
@@ -313,21 +350,24 @@ contexts:
       push: set
     - match: (?i:\bvalues\b)
       scope: keyword.other.DML.II.sql
+    - include: distinct
     - include: joins
-    - match: \b(?i:distinct)\b
-      scope: keyword.other.DML.sql
     - match: \b(?i:group\s+by|order\s+by|having|where)\b
       scope: keyword.other.DML.sql
     - match: \b(?i:from)\b
       scope: keyword.other.DML.sql
-      push: maybe-subquery
+      push: table-name-or-subquery
     - match: (?i)\b(asc|desc)\b
       scope: keyword.other.order.sql
 
+  distinct:
+    - match: \b(?i:distinct)\b
+      scope: keyword.other.DML.sql
+
   joins:
-    - match: (?i)\b(?:inner|(?:full\s+)?outer|cross|left|right)\s+join\b
+    - match: (?i)\b(?:inner|(?:full|left\s+)?outer|cross|left|right)\s+join\b
```

Comment:
T-SQL needs to add `APPLY` to the list of `JOIN`s.

Reply:
Also, bare `JOIN` is okay, right? It implies `INNER JOIN`?

Reply:
I believe I have fixed this, though I haven't added syntax tests for it yet